usage-examples/go/atlas-sdk-go/scripts/bluehawk.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+usage() {
+  cat <<EOF
+Usage: $(basename "$0") <command> [flags]
+
+Commands:
+  snip   Extract code examples from Bluehawk snippets
+  copy   Copy sanitized project files
+
+Example (non‐interactive):
+  $(basename "$0") copy --ignore="tests/*.go" --rename='{"old.md":"new.md"}'
+EOF
+  exit 1
+}
+
+# defaults
+CMD=""
+PROJECT=$(git rev-parse --show-toplevel)
+INPUT_DIR="$PROJECT/usage-examples/go/atlas-sdk-go"
+OUTPUT_DIR="$PROJECT/generated-usage-examples/go/atlas-sdk-go/"
+STATE=""
+IGNORE_PATTERNS=(
+  "README.md"
+  "scripts/"
+  "tests/"
```

Comment:
Nit: currently, there are no `tests/` dirs in this dir, so I'm guessing this is leftover from a prior project structure and can be removed from this script.